dataland-frontend/src/components/resources/frameworkDataSearch/ShowReportsBanner.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="col-12 text-left bg-white mb-4 mp-0 all-sides-blue-border" style="width: 99%">
+    <p class="font-bold text-gray-800 mt-0 mb-0 mp-0 ml-0 mr-0" style="font-size: 11pt">
+      Data extracted from the company report. Company reports:
+    </p>
+    <p id="reportList">
+      {{ resetReportsCount() }}
+      <span v-for="[name, rep] in Object.entries(reports)" :key="name">
+        {{ reportPlus() }}
+        <span
+          @click="downloadReport(rep.reference)"
+          class="font-semibold underline text-primary cursor-pointer"
+          data-test="Report-Download"
+          >{{ name }}</span
+        >
+        <span v-if="reportCounter < Object.keys(reports).length">, </span>
+      </span>
+    </p>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, inject } from "vue";
+import { CompanyReport } from "@clients/backend";
+import { AxiosResponse, AxiosResponseHeaders } from "axios";
+import { ApiClientProvider } from "@/services/ApiClients";
+import { assertDefined } from "@/utils/TypeScriptUtils";
+import Keycloak from "keycloak-js";
+
+export default defineComponent({
+  setup() {
+    return {
+      getKeycloakPromise: inject<() => Promise<Keycloak>>("getKeycloakPromise"),
+    };
+  },
+  name: "ShowReportsBanner",
+  props: {
+    reports: { type: Map<string, CompanyReport> },
+  },
+  data() {
+    return {
+      reportCounter: 10,
+      test: "test",
+      getDocumentsFromStorageProcessed: false,
+      getDocumentsFromStorageResponse: null as AxiosResponse<File> | null,
+      reportContents: [File],
```

Comment:
Das initialisiert ein leeres Array in das Objekte vom Typ "File" sollen, oder was macht das genau?